bindings/python/ML/include/BipedalLocomotion/bindings/ML/velMANN.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file MANN.h
+ * @authors Giulio Romualdi
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_BINDINGS_ML_VEL_MANN_H
+#define BIPEDAL_LOCOMOTION_BINDINGS_ML_VEL_MANN_H
```

Comment:
In case we decide to move to PIMANN we can change the name of this include guard